website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -261,11 +270,10 @@ For compatibility, both the dbt Cloud CLI and dbt Core are invoked by running <c
 
 If you have dbt Core installed locally, either:
 
-1. Install using [pip](/docs/cloud/cloud-cli-installation?install=pip#install-dbt-cloud-cli).
-
-2. Install natively, but ensure that you deactivate your Python environment or uninstall it using `pip uninstall dbt` before proceeding.  
+- Install using the <code>pip3 install dbt</code> [pip](/docs/cloud/cloud-cli-installation?install=pip#install-dbt-cloud-cli) command, or
+- Install natively, however ensure you deactivate your Python environment or uninstall it using `pip uninstall dbt` before proceeding.  
```

Comment:
"Install natively, ensuring that you either deactivate the virtual environment containing core or create an alias for the cloud cli."

It seems "creating an alias for the cloud cli" needs its own section?